schemas/os.yml

```
@@ -13,6 +13,20 @@
   type: group
   fields:
 
+    - name: type
+      level: extended
+      type: keyword
+      short: 'Which commercial OS family (one of: linux, macos, unix or windows).'
+      description: >
+        Use the `os.type` field to categorize the operating system in one of
+        the broad commercial families.
+
+        One of these following values should be used (lowercase): linux, macos, unix, windows.
+
+        If the OS is not part of any of this list, the field should not be populated.
```

```suggestion
        If the OS is not part of any family on this list, the field should not be populated.
```

Minor adjustment suggestion

---

Still not a fan of the sentence here. Let's see if we can reword this some more.